tensorflow/compiler/mlir/xla/transforms/map_xla_to_scalar_op.h

```diff
@@ -119,8 +119,23 @@ template <>
 inline Value MapLhloOpToStdScalarOp<xla_lhlo::AbsOp>(
     Location loc, ArrayRef<Type> result_types, ArrayRef<Value> args,
     OpBuilder* b) {
-  return MapLhloOpToStdScalarOpImpl<FloatType, ::mlir::AbsFOp>{}(
-      loc, result_types, args, b);
+  Type element_type = args.front().getType();
+  if (element_type.isa<FloatType>()) {
+    return MapLhloOpToStdScalarOpImpl<FloatType, ::mlir::AbsFOp>{}(
+        loc, result_types, args, b);
+  }
+  if (element_type.isa<IntegerType>()) {
+    const auto& lhs = args[0];
```

Comment:
Nit: `Value lhs = args[0]`

Also can you add a comment on the expansion at the beginning of the block? Something like:
```
// lhlo.abs(x, y) ->  select((x>0), x, (0-y))
```

Reply:
fixed